diff --git a/node_modules/@capacitor-community/bluetooth-le/android/build.gradle b/node_modules/@capacitor-community/bluetooth-le/android/build.gradle
index 8ed74d4..60c7849 100644
--- a/node_modules/@capacitor-community/bluetooth-le/android/build.gradle
+++ b/node_modules/@capacitor-community/bluetooth-le/android/build.gradle
@@ -35,7 +35,7 @@ android {
     buildTypes {
         release {
             minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
     lintOptions {
